client/.gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
do we need this file?

Reply:
@ciremusyoka you are right...seems like we do not need it